src/welcome-message.ts

```diff
@@ -0,0 +1,76 @@
+import { LitElement, html, css, CSSResult } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+const mailToString = `mailto:donations@archive.org?subject=Please%20Confirm%20my%20Monthly%20Giving%20Circle%20Status&body=Dear%20Philanthropy%20Team%2C%0A%0AI%20am%20already%20a%20member%20of%20the%20Monthly%20Giving%20Circle%2C%20but%20I%20do%20not%20see%20my%20recurring%20donation%20under%20Account%20Settings.%20Please%20use%20the%20following%20information%20to%20locate%20my%20donation%3A%0A%0AFull%20name%3A%20____________%0A%0AEmail%20address(es)%3A%20____________%0A%0A*Please%20provide%20any%20alternative%20email%20addresses%20you%20may%20have%20used%20to%20create%20your%20recurring%20donation.%0A%0AIf%20you%20have%20access%20to%20a%20monthly%20donation%20receipt%2C%20please%20include%20a%20screenshot%20with%20your%20email.%0A%0AThanks!`;
```

Comment:
Non-blocking suggestion: To make this more readable, you could write this out non-escaped with line breaks and everything and use `encodeURI` to encode it, ie:

```
const mailToString = `mailto:donations@archive.org?subject=Please Confirm my Monthly Giving Circle Status&body=Dear Philanthropy Team,

I am already a member of the Monthly Giving Circle, but I do not see my recurring donation under Account Settings. Please use the following information to locate my donation:

Full name: ____________

...`;
```

then

```
const encoded = encodeURI(mailToString);
// encoded: mailto:donations@archive.org?subject=Please%20Confirm%20my%20Monthly%20Giving%20Circle%20Status&body=Dear%20Philanthropy%20Team,%0A%0AI%20am%20already%20a%20member%20of%20the%20Monthly%20Giving%20Circle,%20but%20I%20do%20not%20see%20my%20recurring%20donation%20under%20Account%20Settings.%20Please%20use%20the%20following%20information%20to%20locate%20my%20donation: ...
```

Reply:
ok. i'll circle back on this one